crates/ty_python_semantic/resources/mdtest/mro.md

```diff
@@ -288,26 +288,58 @@ reveal_type(Z.__mro__)  # revealed: tuple[<class 'Z'>, Unknown, <class 'object'>
 
 ## `__bases__` lists with duplicate bases
 
+<!-- snapshot-diagnostics -->
+
 ```py
-class Foo(str, str): ...  # error: 16 [duplicate-base] "Duplicate base class `str`"
+from typing_extensions import reveal_type
+
+class Foo(str, str): ...  # error: [duplicate-base] "Duplicate base class `str`"
 
 reveal_type(Foo.__mro__)  # revealed: tuple[<class 'Foo'>, Unknown, <class 'object'>]
 
 class Spam: ...
 class Eggs: ...
+class Bar: ...
+class Baz: ...
+
+# fmt: off
+
+# error: [duplicate-base] "Duplicate base class `Spam`"
+# error: [duplicate-base] "Duplicate base class `Eggs`"
 class Ham(
     Spam,
     Eggs,
-    Spam,  # error: [duplicate-base] "Duplicate base class `Spam`"
-    Eggs,  # error: [duplicate-base] "Duplicate base class `Eggs`"
+    Bar,
+    Baz,
+    Spam,
+    Eggs,
 ): ...
```

Comment:
Could you add an example where we suppress the diagnostic. Just to make sure it works as expected when suppressing it on line 309 or 316

Reply:
added in https://github.com/astral-sh/ruff/pull/17914/commits/e99571bfb2d4faa55322670d21644ed1ccd83a58